doctr/export_as_pdfa.ipynb

```diff
@@ -13,7 +13,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 9,
```

Comment:
That should be the first cell (or null)  (cells are supposed to be executed in the correct order, so at worst, it should the first cell to be executed